app/models/membership.rb

```diff
@@ -8,10 +8,17 @@
 #  send_email      :boolean
 #  created_at      :datetime         not null
 #  updated_at      :datetime         not null
+#  org_admin       :boolean
 #
 
 class Membership < ActiveRecord::Base
   belongs_to :user
   belongs_to :organization
   has_many :matches
+
+  delegate :name, to: :user, allow_nil: true
```

Comment:
Optional, but you can clean some of this up with [with_options](http://edgeguides.rubyonrails.org/active_support_core_extensions.html#with-options) I think? Also, [delegate](http://edgeguides.rubyonrails.org/active_support_core_extensions.html#method-delegation) can take multiple methods on a single line (`delegate :name, :email, to: :user`)